lineman/spec-e2e/inbox_spec.coffee

```diff
@@ -13,3 +13,8 @@ describe 'Inbox Page', ->
 
     expect(inboxHelper.anyThreads()).not.toContain('Muted discussion')
     expect(inboxHelper.anyThreads()).not.toContain('Old discussion')
+
+  it 'can mark all threads in a group as read', ->
```

Comment:
I've been pushing for a new way of writing our e2es.

Using the PageHelper module, you could rewrite this as:
```
page.expectElement('.inbox-page__group')
page.click('.inbox-page__mark-as-read')
page.expectNoElement('.inbox-page__group')
```